src/registration/terms-of-service/TermsOfServicePage.test.ts

```diff
@@ -71,6 +76,13 @@ const setupMockAjax = async (
     registeredAt: new Date('1970-01-01'),
     updatedAt: new Date('1970-01-01'),
   };
+  const mockSamUserCombinedState: SamUserCombinedStateResponse = {
+    samUser: mockSamUserResponse,
+    terraUserAllowances,
+    terraUserAttributes: { marketingConsent: false } as SamUserAttributes,
```

Comment:
Unnecessary cast can be removed.

```suggestion
    terraUserAttributes: { marketingConsent: false },
```